src/coreclr/jit/targetamd64.h

```diff
@@ -132,15 +142,19 @@
   #define RBM_INT_CALLEE_SAVED    (RBM_EBX|RBM_ESI|RBM_EDI|RBM_ETW_FRAMED_EBP|RBM_R12|RBM_R13|RBM_R14|RBM_R15)
   #define RBM_INT_CALLEE_TRASH    (RBM_EAX|RBM_ECX|RBM_EDX|RBM_R8|RBM_R9|RBM_R10|RBM_R11)
   #define RBM_FLT_CALLEE_SAVED    (RBM_XMM6|RBM_XMM7|RBM_XMM8|RBM_XMM9|RBM_XMM10|RBM_XMM11|RBM_XMM12|RBM_XMM13|RBM_XMM14|RBM_XMM15)
-  #define RBM_FLT_CALLEE_TRASH    (RBM_XMM0|RBM_XMM1|RBM_XMM2|RBM_XMM3|RBM_XMM4|RBM_XMM5)
+  /* NOTE: Sync with variable name defined in compiler.h */
+  #define RBM_FLT_CALLEE_TRASH_INIT (RBM_XMM0|RBM_XMM1|RBM_XMM2|RBM_XMM3|RBM_XMM4|RBM_XMM5)
+  #define RBM_FLT_CALLEE_TRASH    rbmFltCalleeTrash
 #endif // !UNIX_AMD64_ABI
 
   #define RBM_OSR_INT_CALLEE_SAVED  (RBM_INT_CALLEE_SAVED | RBM_EBP)
 
   #define REG_FLT_CALLEE_SAVED_FIRST   REG_XMM6
   #define REG_FLT_CALLEE_SAVED_LAST    REG_XMM15
 
-  #define RBM_CALLEE_TRASH        (RBM_INT_CALLEE_TRASH | RBM_FLT_CALLEE_TRASH)
+  #define RBM_CALLEE_TRASH_INIT   (RBM_INT_CALLEE_TRASH | RBM_FLT_CALLEE_TRASH)
```

Comment:
Isn't there a way to avoid defining all these variables?
Can't you just define the three global variables instead and use them here? See https://github.com/kunalspathak/runtime/commit/08d15493943212d0846118111f54e1719a94763e

Reply:
I defined them like that for consistency, but yes I agree for performance your suggestion works much better. I've merged in.